### PR TITLE
DFPL-1240: Preparing the scheduler for production

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-1240

### Change description ###
Preparing the scheduler for production

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
       jobKind: CronJob
     job:
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:prod-7cc7082-20230302152355 #{"$imagepolicy": "flux-system:fpl-cron-ccd-data-migration-tool"}
+      image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:pr-101-cc746c7-20230317112724 #{"$imagepolicy": "flux-system:fpl-cron-ccd-data-migration-tool"}
```
Comment (thomast1906) [inline apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml:12]: a pr-image in production?

Comment (thomast1906) [reply apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml:12]: CI check will fail with current request of` pr-*` image in production

**Review by thomast1906 — CHANGES_REQUESTED** [review]
pr-* image in Production? Not usually allowed - please review.

File: apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml (hunk 1 above)
Comment (SomeshRA) [reply apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml:12]: Updated to prod image - prod-7cc7082-20230302152355

File: apps/family-public-law/prod/base/kustomization.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../base
+  - ../../fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yam
```
[the remark this answers is not in the document]
Comment (SomeshRA) [reply apps/family-public-law/prod/base/kustomization.yaml]: Corrected the typo

**Review by thomast1906 — CHANGES_REQUESTED** [review]
ALot of duplicate values in 00.yaml & 01.yaml, please move duplicated values to the base file.

**Comment by jonsoloway-hmcts** [comment]
I can't see the CPU/Memory changes that were performance tested applied to the Production configuration

**Comment by SomeshRA** [comment]
> I can't see the CPU/Memory changes that were performance tested applied to the Production configuration
I has been moved to the base file
https://github.com/hmcts/cnp-flux-config/pull/21667/files#diff-6c1376bc94aa46f7183ff1a71d8320c7cd14df8cfeda041c10c32103429a7ad6R13

File: apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml [hunk 3]
```diff
@@ -10,6 +10,17 @@ spec:
     job:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:prod-f597322-20230321171332 #{"$imagepolicy": "flux-system:fpl-cron-ccd-data-migration-tool"}
+      schedule: "30 12 30 2 *"
+      cpuRequests: "1000m"
+      cpuLimits: "2500m"
+      memoryRequests: "1024Mi"
```
Comment (thomast1906) [inline apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml:16]: THis is going into all environments yes?

Comment (SomeshRA) [reply apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml:16]: Demo doesn't need this.

Comment (thomast1906) [reply apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml:16]: You would need to create some new patch files per environment 

example; https://github.com/hmcts/cnp-flux-config/tree/master/apps/cnp/plum-frontend

& kustomized to the likes of AAT - like this, https://github.com/hmcts/cnp-flux-config/blob/master/apps/cnp/aat/base/kustomization.yaml#L8

Comment (SomeshRA) [reply apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml:16]: I have changed as per shared example.

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**